randovania/games/dread/json_data/Cataris.txt

```diff
@@ -2670,6 +2683,7 @@ Extra - asset_id: collision_camera_031
       Any of the following:
           Spider Magnet or Space Jump
           Flash Shift and Grapple Beam
```

Comment:
Do actually implement the changes before marking as resolved. I'm sure Grapple alone is fine, and Spin Boost is enough to do this.

Reply:
Again apologies for the things that didn't actually get changed on my end, shouldn't need to update source part way through to add a new resource any time soon